Set CMake minimum version and remove duplicate doc directory which violates
CMP0013

--- CMakeLists.txt.orig	2015-03-23 21:04:09 UTC
+++ CMakeLists.txt
@@ -1,5 +1,7 @@
 project(krecipes)
 
+cmake_minimum_required(VERSION 2.8.9)
+
 set(CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake 
                       ${CMAKE_MODULE_PATH})
 
@@ -21,7 +23,6 @@ add_subdirectory( pics )
 add_subdirectory( data )
 add_subdirectory( layouts )
 add_subdirectory( mimetypes )
-add_subdirectory( doc )
 
 include(MacroOptionalAddSubdirectory)
 macro_optional_add_subdirectory( po )
